plugins/ILibPlugin/index.js

```diff
@@ -169,11 +169,12 @@ class ILibPlugin {
 				if (pkgName.indexOf('@enact') === 0) {
 					this.options.create = false;
 				}
-				if (pkgName === '@enact/i18n') {
-					this.options.ilib = 'ilib';
-				} else {
-					this.options.ilib = packageSearch(process.cwd(), '@enact/i18n/ilib');
-				}
+				// look for ilib as a root-level node_module package location
+				this.options.ilib =
+					// Backward compatability for old Enact libraries
+					packageSearch(process.cwd(), path.join('@enact', 'i18n', 'ilib')) ||
+					packageSearch(process.cwd(), 'ilib') ||
+					(pkgName === '@enact/i18n' && fs.existsSync(path.join(process.cwd(), 'ilib')) && 'ilib');
```

Comment:
Do we need a warning here if `this.options.ilib` is empty?  Does the catch description below need to be updated?

Reply:
Falsey/empty/undefined `this.options.ilib` is valid possibility and we handle that as "ilib not found" (which that would indicate) later in the plugin. The throw error is an issue however and I'll push an update.